src/ioctls/vm.rs

```diff
@@ -193,6 +193,55 @@ impl VmFd {
         }
     }
 
+    /// Directly injects a MSI message as per the `KVM_SIGNAL_MSI` ioctl.
```

Comment:
Please split documentation comment in summary and description. The summary should be followed by an empty comment line.

Reply:
done